frontend/pages/group/data/recipes.vue

```diff
@@ -337,7 +363,21 @@ export default defineComponent({
 
       const { response, data } = await api.bulk.bulkSetSettings({ recipes, settings: recipeSettings });
```

Comment:
```suggestion
      await api.bulk.bulkSetSettings({ recipes, settings: recipeSettings });
```

[sand] unused variables